mnist-classifaction/train_job/README.md

```diff
@@ -0,0 +1,49 @@
+# getting-started-examples
+Examples to get started with using TrueFoundry
+
+Deployment
+---
+This example runs a simple mnist-classifaction job.
+Mainly this example shows how to deploy to TrueFoundry using a Pythonfile and TrueFoundry Python SDK.
+
+## Run Locally
+
+1. Install requirements
+
+```shell
+python -m pip install -r requirements.txt
+```
+
+2. Start the app
```

Comment:
Create a new section setup and move 
"Create Ml Repo and Grant Editor access to that"

This section of run local should just have installing requirements and running

Reply:
do this at other places too